packages/admin-panel/src/VizBuilderApp/api/constants.js

```diff
@@ -7,6 +7,5 @@
 export const DEFAULT_REACT_QUERY_OPTIONS = {
   retry: 0,
   // should be refetched in the background every hour
-  staleTime: 1000 * 60 * 60,
```

Comment:
The query cache has caused some annoying scenarios such as when you update the viz from somewhere else (Admin Panel Dashboard Item modal) and it doesn't reflect in the viz-builder page. So I think it's better to disable the cache and always fetch the latest data

Reply:
Good call - viz builder data should be fairly quick to fetch!